atoMEC/xc.py

```diff
@@ -1,19 +1,20 @@
 """
-The xc module handles everything connected to the exchange-correlation energy and potential.
+The xc module handles everything connected to the exchange-correlation energy and
+potential.
```

Comment:
We again have failed checks. Can you think of a short module summary line? A la

```python
"""Short xc module summary line.

The xc module handles everything connected to the exchange-correlation energy and
potential.

...
"""
```

Reply:
Argh, ok should be corrected now